annif/project.py

```diff
@@ -153,6 +155,16 @@ def vocab(self):
     def subjects(self):
         return self.vocab.subjects
 
+    @property
+    def is_trained(self):
+        return bool(glob(os.path.join(self._datadir_path, '*')))
+
+    @property
+    def modification_time(self):
```

Comment:
Same comment as for `is_trained` above - I think it makes sense to implement this in individual backends instead of here on the project level, for the same reasons.